.jenkins/Jenkinsfile

```diff
@@ -1,32 +1,37 @@
 @Library("OpenEnclaveCommon") _
 oe = new jenkins.common.Openenclave()
+GLOBAL_TIMEOUT = 240
```

Comment:
Just one small nit on this one. TBH I have no idea what 240 is measuring. Mind adding a comment to inform users if this is in seconds/minutes?

Reply:
I have addressed with a comment in all Jenkinsfiles

Reply:
Good idea! @vlaza please specify that it's minutes everywhere.